flat/frontend-rules.mjs

```diff
@@ -4,7 +4,7 @@ export default {
     eqeqeq: ['error', 'smart'],
     'no-alert': ['error'],
     'no-caller': ['error'],
-    'no-console': ['error'],
+    'no-console': ['error', { allow: ['warn', 'error'] }],
```

Comment:
I'm happy OK this. I thought we normally use the logger for this [Example](https://github.com/oat-sa/vite-app-template/blob/test/eslint-config-flat4/src/core/apiRequest.js#L50)